motr/client.h

```diff
@@ -896,6 +904,75 @@ struct m0_config {
 	m0_bcount_t mc_addb_size;
 };
 
+/* Constants for protection info type, max types supported is 8 */
+enum
+{
+	M0_PI_TYPE_MD5,
+	M0_PI_TYPE_MD5_INC_DIGEST,
+	M0_PI_TYPE_CRC,
+	M0_PI_TYPE_MAX
+};
+
+enum m0_pi_calc_flag {
+
+	/* No special handling */
+	M0_PI_CALC_NONE = 0,
+	/* PI calculation for data unit 0 */
+	M0_PI_CALC_UNIT_ZERO = 1 << 0
+};
+
+M0_BASSERT(M0_PI_TYPE_MAX <= 8);
+
+struct m0_pi_hdr {
+	/* type of protection algorithm being used */
+	uint8_t pi_type : 3;
+        /*size of PI Structure in multiple of  32 bytes*/
```

Comment:
Alignment seems to be off here, please check and correct.

Reply:
Done